teamcity-rest-client-impl/src/test/kotlin/org/jetbrains/teamcity/rest/BuildTest.kt

```diff
@@ -146,4 +146,24 @@ class BuildTest {
             assertNotNull(iterator.next())
         }
     }
+
+    @Test
+    fun test_parameters() {
```

Comment:
Actually this test doesn't work - I don't understand why no one noticed that parameters don't work. Though I don't know how to fix it.

Reply:
Hmm, it's strange, the parameters are properly returned in xml response but indeed are missing in json response. Could you please ask TeamCity guys about that?

Reply:
I've got it: guest auth here isn't enough (just try it in incognito mode) - I'll add auth to this test.